circularSeekBar/src/main/java/me/tankery/lib/circularseekbar/CircularSeekBar.kt

```diff
@@ -705,6 +747,12 @@ class CircularSeekBar @JvmOverloads constructor(
         pointerPaint.strokeJoin = Paint.Join.ROUND
         pointerPaint.strokeCap = circleStyle
 
+        pointerFillPaint.set(pointerPaint)
+        pointerFillPaint.color = pointerFillColor
+        pointerFillPaint.style = Paint.Style.FILL_AND_STROKE
+        pointerFillPaint.strokeWidth =
+            (pointerStrokeWidth - pointerFillInnerStrokeWidth).coerceAtLeast(0f)
```

Comment:
If we use a concept like "pointerInnerStrokeWidth" or "pointerOverlayStrokeWidth", we could simply apply the pointerInnerStrokeWidth here, and let the developer to do the math, and give developer the flexibility.